kernel/syscalls/gettid.rs

```diff
@@ -0,0 +1,9 @@
+use crate::{process::current_process, result::Result, syscalls::SyscallHandler};
+
+impl<'a> SyscallHandler<'a> {
+    /// returns the caller's thread ID (TID).  
+    /// In a single-threaded process, the thread ID is equal to the process ID (PID)
+    pub fn sys_gettid(&mut self) -> Result<isize> {
+        Ok(current_process().pid().as_i32() as isize)
+    }
```

Comment:
As you noted in the PR's description, I think the implementation of gettid should be moved into `Process::tid()`.

Reply:
Also, if cite a sentence from the man page, please add a link to it.

Reply:
Since logic moved to `Process::tid()` comment in impl is no longer necessary. But, maybe it is a good idea to add links to man pages.